shell/edit/provisioning.cattle.io.cluster/tabs/registries/RegistryConfigs.vue

```diff
@@ -148,6 +148,7 @@ export default {
             />
 
             <SelectOrCreateAuthSecret
+              :key="row.value.hostname"
```

Comment:
Is `hostname` really unique? I seem to be able to reproduce the behavior described in the issue if I have a form with two `a.registry.com` entries.

Reply:
Ideally UI should run some validation against having unique registry host names, currently UI allows to have multiple registries with the same host name when you're editing, but once you save it, it merges them into one.

Reply:
I made it more unique by combining the host name with secret name, but I'll create a separate issue later for adding validation after discussing it with the team

Reply:
Thanks! I think this is better and the situation should improve with better validation down the line.